common/config/rush/pnpm-lock.yaml

```diff
@@ -682,6 +688,15 @@ packages:
     resolution: {integrity: sha512-k2Ty1JcVojjJFwrg/ThKi2ujJ7XNLYaFGNB/bWT9wGR+oSMJHMa5w+CUq6p/pVrKeNNgA7pCqEcjSnHVoqJQFw==}
     dev: false
 
+  /@gear-js/api/0.24.0_@polkadot+api@7.15.1:
+    resolution: {integrity: sha512-rhHqxhqNFJYPiX36liQV1lPYUzd0tdftH+ldkKRLdkRtBqeEcJjNo0rIMEGRxOSjs9nFtCFZKPRekonahhyYaQ==}
+    peerDependencies:
+      '@polkadot/api': ^8.11.3
```

Comment:
Here we go again :( 
Is there any way to make this dependency optional only if the gear support is required? `pokadot/api` is really huge and including it as a dependency will make our squid images more heavy + it's know to be very volatile and not very stable.